crates/unftp-sbe-gcs/src/uri.rs

```diff
@@ -1,210 +0,0 @@
-use hyper::Uri;
-use libunftp::storage::{Error, ErrorKind};
-use percent_encoding::{utf8_percent_encode, NON_ALPHANUMERIC};
-use std::path::{Path, PathBuf};
-
-#[derive(Clone, Debug)]
-pub(crate) struct GcsUri {
```

Comment:
Curious why you would combine the GcsUri stuff with the client if it was already separate? what is your thinking?

Reply:
I wanted to move all HTTP related code to one place, so I merged the request logic with GcsUri, and then renamed GcsUri to GcsClient. I never really liked the way I set up GcsUri in the first place - I think I created the wrong abstraction then.

If you do think this is a valuable abstraction to have, I can split the URI construction from the request logic. I don't think this is common practice, though.

Reply:
No lets leave it as you have it now.